net/uradvd/src/Makefile

```diff
@@ -0,0 +1,4 @@
+all: uradvd
+
+uradvd: uradvd.c
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -Wall -o $@ $^ $(LDLIBS) -lrt
```

Comment:
You need to delete the `src` directory, as you are using a proper source repo now.